03-rec-poly/homework-03/hw-03.hs

```diff
@@ -0,0 +1,16 @@
+import Data.Maybe
+
+--Exercise 1
+skips :: [a] -> [[a]]
+skips xs = filter (catMaybes (zipWith (filterIdx) (take (length xs) (repeat (zip [1..] xs))) [1..]))
```

Comment:
Few tips to make this even golfier :)
You can get rid of repeat (use fmap instead of zipWith)
You can get rid of take if you place length xs somewhere else
You might want to use mapMaybe

Reply:
Create a new PR for this @peironggg